.github/workflows/nrn-modeldb-ci.yaml

```diff
@@ -80,7 +80,14 @@ jobs:
     - name: Set models to run for PR
       run: |
         # Select a few models to run CI for PRs 
-        echo 'MODELS_TO_RUN=279 3264 22203 144549 105507 194897' >> $GITHUB_ENV
+        #   64195   -> run: null
+        #   22203   -> skip: true
+        #   156120  -> several model directories
+        #   97868   -> failed nrnivmodl
+        #   146030  -> curate_patterns
+        #   244679  -> no entry in modeldb-run.yaml
+        #   rest:   -> currently yielding differences in the pipeline
+        echo 'MODELS_TO_RUN=64195 22203 156120 51781 97868 22203 97756 97917 105507 136803 138379 138382 146030 244679 251881' >> $GITHUB_ENV
```

Comment:
@alexsavulescu : does this mean we are now running more models on CI triggered via PRs? (the above description per model seems like they are with errors but other part of descriptions says "MODELS TO RUN". So bit confusing here)

Reply:
For PRs we only have a few models for quick sanity checking. As per the comments above, we are targeting different features (since we haven't worked on unit testing yet -> #38 ). The idea is that one could still run all models via the manual workflow to make sure there is no regression before merging a PR (which we do systematically). 

I do agree there are much more improvements to be made.

Reply:
> quick sanity checking.

(quick means a few minutes compared to ~45 minutes for full run)